lib/prototype-admin/password.html

```diff
@@ -0,0 +1,64 @@
+
+{% extends "layout_unbranded.html" %}
+
+{% set serviceName %}
+{% endset %}
+
+{% block pageTitle %}
+  {% if error == "wrong-password" %}
+    Error:
+  {% endif %}
+  Sign in - GOV.UK Prototype Kit
+{% endblock %}
+
+{% block scripts %}
+  <!-- We don't need any scripts -->
+{% endblock %}
+
+{% block content %}
+
+<form method="post">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% if error == "wrong-password" %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: [
+            {
+              text: "The password is not correct",
+              href: "#password"
+            }
+          ]
+        })}}
+      {% endif %}
+
+      <h1 class="govuk-heading-xl">
+        This is a prototype used for research
+      </h1>
+
+      <p>
+          It is not a real service. You should only continue if you have been invited to test this prototype.
+      </p>
+
+      {{ govukInput({
+        name: "password",
```

Comment:
Should this input be a password?  Later we can use show/hide password functionality.